src/Optimizations/itensor_network.jl

```diff
@@ -0,0 +1,26 @@
+using ChainRulesCore
+using ..ITensorNetworks
+using ..ITensorNetworks: split_links
+
+inv_op(::typeof(addtags)) = :removetags
```

Comment:
Couldn't this return the function `removetags` instead of the Symbol `:removetags`?

Reply:
We could just move these definitions to the `ITensorChainRules` module.

Reply:
yes just returning the function is better.